### PR TITLE
Improves *forced* speech created by brain trauma to be within RP standards.

### PR DESCRIPTION
**Pull request by Haliris**
## About The Pull Request
Changes most sentences some brain traumas forced your character to say, as some were LRP or would straight up lead you to break the rules. Also had to change derpspeech as it would replace "you" with "u" randomly, making players netspeak.

## Why It's Good For The Game
The OOC nature of the majority of these sentences made them essentially impossible to engage with, while making it obvious that the person had a brain trauma. 
In the case in which the players wouldn't assume a brain trauma, they'd report the affected player for breaking the server rules.

The main objective of this PR is to make it so that brain traumas don't banbait players anymore.
I tried to write replacement sentences that had the possibility of creating engagement with the players, for others to react to and to pick up on. 

## Testing Photographs and Procedure

<details>

<summary>Screenshots&Videos</summary>

image

They load.
</details>

## Changelog
:cl:
tweak: Changed and improved some strings used by traumas for forced speech to make them more in line with RP standards.
/:cl:

### CODE REVIEW

File: strings/traumas.json [hunk 1]
```diff
@@ -1,112 +1,118 @@
...
-		"@pick(semicolon)this SI mY stATIon......",
-		"who the HELL do u thenk u r?!!!!",
-		"geT THE FUCK OUTTTT",
-		"H U G B O X",
-		"@pick(semicolon)CRASHING THIS STTAYTION WITH NIO SURVIVROS",
-		"@pick(servers) is down!! @pick(bug)",
+		"@pick(semicolon)I'M A PONY, NEEEEEEIIIIIIIIIGH",
+		"I love you so much...",
+		"@pick(semicolon)MEDBAY TO ME!! I AM DEAD!!",
+		"@pick(semicolon)@pick(role) is KILLING me HELP!!!",
```
Comment (Penwin0) [inline strings/traumas.json:6]: @pick(role) won't work, as it's not defined like semicolon is, search for semicolon in that same file

**Review by Penwin0 — CHANGES_REQUESTED** [review]

**Review by EvilDragonfiend — APPROVED** [review]
I've hated this so much

**Comment by WhereAmO** [comment]
Does this also affect the Spray Tan overdose messages?

**Comment by Haliris** [comment]
It doesn't affect the messages played by spray tan overdose, those are defined in the overdose code itself.

**Review by Rukofamicom — APPROVED** [review]

**Comment by Rukofamicom** [comment]
Thank you for your service in making this.

**Review by Penwin0 — APPROVED** [review]

**Pull request merged**